Assignment2/Cristinalosno29-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,41 @@
+{
+    "@context":{
+        "vocab":"http://example.org/elements/",
+        "name":"vocab:name",
+        "hometown":"vocab:hometown",
+        "age":"vocab:age",
+        "almaMater":"vocab:almaMater",
+        "parentsName":"vocab:parentsName",
+        "subjects":"vocab:subjects",
+        "name":"subjects:name",

```

Comment:
You have a duplicate name. Rename it, maybe as "subject":"vocab:subject" according to the name provided at the structure of your subject's array.

Reply:
Still pending...

Reply:
The "subject" attribute is not defined. Please, add it.